part_1/title_IX/sekcja_57_PL.md

```diff
@@ -14,25 +14,40 @@ posiadaczowi pozwolenia na broń lub listu przewozowego zostało przedstawione o
+a) w przypadku osoby fizycznej ściganej za przestępstwo, o którym mowa w § 22 ust. 1,
+(B) w przypadku osoby prawnej, ścigania karnego za przestępstwo umyślne,
+c) w przypadku osoby fizycznej postępowanie administracyjne dotyczące przestępstw popełnionych przez
+1. posiada, przechowuje lub nosi broń z naruszeniem § 8,
+2. niezgodnie z przepisami sekcji 20a ust. 3 nie podlega badaniu lekarskiemu lub nie jest przedstawiony
+Sprawozdanie o stanie zdrowia właściwego departamentu policji,
+3. Fotografowanie z broni z naruszeniem art. 28 ust. 5
+4. wbrew § 29 ust. 1 (L) nie poddawaj się badaniu w celu stwierdzenia, czy nie
+Wpływ na alkohol lub inną substancję uzależniającą lub z naruszeniem art. 29 ust. 3
+(B) nosić broń lub manipulować bronią publicznie lub publicznie
+Mimo że jego zdolność do tego zmniejsza się dzięki użyciu alkoholu
+Napoje, stosowanie innych substancji uzależniających, leków lub w wyniku choroby,
+5. nie postępuje zgodnie z art. 41 ust. 5, lub
+6) nie powiadomi o nabyciu prawa własności broni palnej kategorii A, B lub C zgodnie z sekcją 42 (1)
+Or
+(D) w przypadku osoby prawnej lub fizycznej prowadzącej działalność gospodarczą, postępowania administracyjnego za przestępstwo
+Zaangażowanie w posiadanie lub posiadanie broni lub amunicji z naruszeniem sekcji 8.
+(2) Odpowiedni wydział policja może podjąć decyzję o zapewnieniu kategorii A, B, C lub D, amunicji, prochu czarnego polowania, proch bezdymny, zapałki, licencja pistolet, broń licencji pistolety certyfikatów, broni palnej list przewozowy dla stałych eksportu stały import lub tranzyt broni lub amunicji, pozwolenia na wywóz na podstawie ustawy o kontroli handlu produktami, których posiadanie w Republice Czeskiej ze względów bezpieczeństwa, Europejską kartę broni palnej, zezwolenie na przemieszczanie lub transporty broni i amunicji zgodnie z § 50 i 50a raportowanie, które zostały zajęte przez § 56 i sprawa musi być przedmiotem postępowania na podstawie tego czy specjalna ustawa. Jeżeli w ciągu 1 miesiąca od ich postępowania zatrzymań na podstawie niniejszej ustawy lub sprawy nie jest określany organ uprawniony do prowadzenia postępowania na podstawie specjalnej ustawy, właściwy departament policji postępować zgodnie z pkt 8.
+(3) uczestnikiem postępowania w celu zapewnienia osobie, która kryje w sobie sprawę, o której mowa w ustępie 1 lub 2 i dalszego ich właściciela lub osobę, która jest wymieniona w dokumencie. Osoba rzeczy otrzymano na podstawie ustępu 1 lub 2 posiada, jest on zobowiązany do przekazania tych rzeczy bez zbędnej zwłoki do aresztu policji, który zdecydował o ich bezpieczeństwo lub komunikowania zamiast przechowując je i umożliwić ich przejmowanie przez policję. Policja wyda potwierdzenie przejęcia zabezpieczonych przedmiotów.
+(4) Odwołanie od decyzji zgodnie z ust. 1 i 2 nie ma skutku zawieszającego.
+(5) Jeśli posiadacz broni, amunicji, zakazu uzbrojenia lub dokumentu
```

Comment:
¿posiadacz zakazu uzbrojenia?
Ok, widzę, że jeszcze nie skończyłaś tłumaczenia tego pliku, reszta to chyba surowy _google translate_, więc dalej nie sprawdzam.